src/main/java/cloud/eppo/Utils.java

```diff
@@ -63,6 +70,13 @@ public static String getISODate(Date date) {
     return UTC_ISO_DATE_FORMAT.format(date);
   }
 
+  public static String base64Encode(String input) {
```

Comment:
Android provides built-in higher-level base64 functions, but Java 8 does not, so we roll out own based on the Android implementation.